includes/polling/os/awplus.inc.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+//OS info - https://www.alliedtelesis.com/products/software/AlliedWare-Plus
+//OS Datasheet - https://www.alliedtelesis.com/documents/datasheet-alliedware-plus
+
+//$hardware and $serial use snmp_getnext as the OID for these is not always fixed.
+//However, the first OID is the device baseboard. 
+
+$hardware = trim(snmp_getnext($device, "rscBoardName", "-OQv", "AT-RESOURCE-MIB"), '"');
```

Comment:
The trims here should not be needed snmp_get and snmp_getnext trim surrounding "

Reply:
Done :-)